src/loader.js

```diff
@@ -348,12 +348,16 @@
                             });
                             break;
                         default:
-                            this.cdn("v" + state.__version, function (_req) {
-                                _req(["src/other/Persist"], function (Persist) {
-                                    Persist.create(state, function (widget) {
-                                        callback(widget, _req);
-                                    });
-                                }, requireErrorHandler);
+                            var context = this;
+                            require(['src/common/Utility','src/common/Platform'],function(Utility,Platform){
+                                state = Utility.retrofit_114_serialization(state,new Platform().version());
```

Comment:
The retrofit code, belongs in the loader.js file.

Reply:
This doesn't look correct what if the persist string is v1.14 and the referenced ddl is the correct 1.14 version?